### ISSUE TITLE
[Research Spike] Trial run for error report production

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#2193]
**Description:**
To better estimate the lift required to manually produce error reports for pilot expansion v3 research we need to actually attempt to apply the style guide to a realistic set of errors that might be generated by a quarter of data.

**AC:**

- [x] Set of errors tackled is on the moderate to high side (e.g. ~70 errors)
  - [x] rows 1-15
  - [x] rows 16-30
  - [x] rows 31-45
  - [x] rows 46-60
  - [x] rows 61-76
- [ ] Section 1 error report produced 

**Tasks:**

- [x] Coordinate w/ Alex / DIGIT to pull an appropriately sized transmission report 
- [x] Determine unique error messages
- [x] Generate Header Portion of the file
- [x] Be able to insert errors back into .txt file by case number
- [x] Apply style guide to applicable errors
- [x] Manually tackle remaining errors
- [x] Document a Process that can be reproduced during v3.0 research
- [x] Retro process / end result 

**Supporting Documentation:**
- [Section 1 Instructions](https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_data_reports_tan_ssp_instructions_definitions.pdf)
- [Error Range Style Guide](https://hackmd.io/aPuVeUd0RB-CiBk4WT8nfg)
- [Updated Error Messages](https://teams.microsoft.com/_#/xlsx/viewer/teamsSdk/https:~2F~2Fhhsgov.sharepoint.com~2Fsites~2FTANFDataPortalOFA-UserResearch~2FShared%20Documents~2FUser%20Research~2FRelease%203%20(V3)%20-%20Parity,%20Data%20Parsing,%20Automated%20Status~2FMDTANF_TrRptErrs_FY20tFY21.xlsx?threadId=19:3bb622256c0b4792976dc5f1f4654fca@thread.skype&subEntityId=%257B%2522viewParams%2522%253A%2522id%253D%25252Fsites%25252FTANFDataPortalOFA%25252DUserResearch%25252FShared%252520Documents%25252FUser%252520Research%25252FRelease%2525203%252520%252528V3%252529%252520%25252D%252520Parity%25252C%252520Data%252520Parsing%25252C%252520Automated%252520Status%2526viewid%253D7de5242f%25252Df9d3%25252D4e7f%25252D9747%25252D0f37d8b85078%2522%257D&baseUrl=https:~2F~2Fhhsgov.sharepoint.com~2Fsites~2FTANFDataPortalOFA-UserResearch&fileId=b901fe02-fb73-4db3-98c8-c1bf2182bfe5&ctx=openFilePreview&viewerAction=edit)
- [Reference Guide - Transmission File Demo](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FDesign%2FTransmissionsDemo%2Ehtml&parent=%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FDesign)

**Comment by lilybl1** [comment]
Spoke to Alex about obtaining the transmission error report by COB Tuesday, 10/25. Thank you!

**Comment by lilybl1** [comment]
@reitermb Let's review when you get back. I've stubbed out what a potential "header" could be for the Transmission File. 

Temporary Assistance For Needy Families Program
Transmission Report Section 1 (SSP/TANF) append based on the type of file it is

Date of Transmission: 11/07/22 11:54:44 Calendar Year & Qtr: 2022/04
State/Tribe: {Insert Region #}

State/Tribe Name: {Insert Name}

Case Number Rpt Month Errors


Total Cases Transmitted : XXX
Total Cases Accepted	: XXX
Total Cases Rejected	: X

**Comment by reitermb** [comment]
@ADPennington bumping this into QASP! Given that this was an exploratory ticket for us rather than a deliverable I'm not sure what you'd like QASP review to entail--big picture is that we think we could make manual error report production work on the scale we attempted here (~70 errors). Given that a lot of the grantees we're targeting for the actual 3.0 research have far fewer errors than that we should be in particularly good shape bandwidth wise. 

Also noting that we've made some changes to our idea for the header since Diana's comment (which currently lives in the [New Error Report hack.md](https://hackmd.io/aPuVeUd0RB-CiBk4WT8nfg) aimed at making sure it's a little more consistent with how we're identifying data in TDP than how it was identified in Transmission Reports.
